source/tools.txt

```diff
@@ -1,168 +1,263 @@
-   please let us know or add it back.
+Many developers have written tools for working with {+driver-short+}. If you know
+of a tool for working with MongoDB from Python, please list it here.
+
+.. note::
+  
+   To keep this list current, projects that
+   haven't been updated recently will
+   occasionally be removed from the list or moved to the end.
+
+   If we remove a project that is still being developed or is in active use,
+   please let us know or add it back to the list.
 
 ORM-like Layers
 ---------------
-Some people have found that they prefer to work with a layer that
-has more features than PyMongo provides. Often, things like models and
-validation are desired. To that end, several different ORM-like layers
-have been written by various authors.
-
-It is our recommendation that new users begin by working directly with
-PyMongo, as described in the rest of this documentation. Many people
-have found that the features of PyMongo are enough for their
-needs. Even if you eventually come to the decision to use one of these
-layers, the time spent working directly with the driver will have
-increased your understanding of how MongoDB actually works.
-
-MongoEngine
-  `MongoEngine <http://mongoengine.org/>`_ is another ORM-like
-  layer on top of PyMongo. It allows you to define schemas for
-  documents and query collections using syntax inspired by the Django
-  ORM. The code is available on `GitHub
-  <http://github.com/mongoengine/mongoengine>`_; for more information, see
-  the `tutorial <https://docs.mongoengine.org/tutorial.html>`_.
-
-MincePy
-   `MincePy <https://mincepy.readthedocs.io/en/latest/>`_ is an
-   object-document mapper (ODM) designed to make any Python object storable
-   and queryable in a MongoDB database. It is designed with machine learning
-   and big-data computational and experimental science applications in mind
-   but is entirely general and can be useful to anyone looking to organise,
-   share, or process large amounts data with as little change to their current
-   workflow as possible.
-
-Ming
-  `Ming <https://ming.readthedocs.io/en/latest/>`_ is a
+
+ORM-like (object-relational-mapping-like) layers add features like models and validation to
+{+driver-short+}.
+
+.. tip::
+  
+   We recommend that new {+driver-short+} users begin by working directly with
+   the driver. {+driver-short+} alone will meet most people's needs, and working with
+   it is instructive in how MongoDB works.
+
+- `MincePy <https://mincepy.readthedocs.io/en/latest/>`__ is an
+  object-document mapper (ODM) designed to make any Python object storable
+  and queryable in a MongoDB database. It was designed with machine learning
+  and big-data computational and experimental science applications in mind,
+  but is entirely general and can be useful to anyone looking to organize,
+  share, or process large amounts of data with as little change to their current
+  workflow as possible.
```

Comment:
Suggestion: It's just a really long sentence otherwise.
```suggestion
  and queryable in a MongoDB database. It was designed with machine learning
  and big-data computational and experimental science applications in mind.
  However, it is entirely general and can be useful to anyone looking to organize,
  share, or process large amounts of data with as little change to their current
  workflow as possible.
```